angular/src/lib/sidebar/sidebar.service.ts

```diff
@@ -1,11 +1,15 @@
 import { Injectable } from '@angular/core';
 import { BehaviorSubject } from 'rxjs';
+import { distinctUntilChanged } from 'rxjs/operators';
+
 
 @Injectable()
 export class SidebarService {
 
   private tier = new BehaviorSubject<any>(false);
-  tier$ = this.tier.asObservable();
+  tier$ = this.tier.asObservable().pipe(
+    distinctUntilChanged(),
+  );
```

Comment:
Assuming this is a common pattern to always add distinctUntilChanged() @brspence?

Reply:
Depends on the use case. Your observable may want to react to new events of the same value because it's being orchestrated with other operators. But yeah didn't think that was necessary here